textproc/lilypond-devel/Portfile

```diff
@@ -145,32 +144,35 @@ post-patch {
 # possible to circumvent this issue without rewriting a considerable
 # amount of code.
 
+# g++ uses libstdc++; setting this prevents the application from being
+# flagged as broken.
+configure.cxx_stdlib macports-libstdc++
+
 # blacklist any clang compiler and any old gcc (a better expression is needed)
-compiler.blacklist \
-    *clang* \
-    *llvm-gcc* \
-    *apple-gcc* \
-    gcc \
-    gcc-3.3 gcc-4.0 gcc-4.2
-compiler.whitelist \
-    macports-gcc-8 \
-    macports-gcc-7 \
-    macports-gcc-6 \
-    macports-gcc-5 \
-    macports-gcc-4.9 macports-gcc-4.8 macports-gcc-4.7
-configure.cxx_stdlib    macports-libstdc++
+compiler.blacklist  *clang* \
+                    *llvm-gcc* \
+                    *apple-gcc* \
+                    gcc \
+                    gcc-3.3 gcc-4.0 gcc-4.2
+
+compiler.whitelist  macports-gcc-8 \
+                    macports-gcc-7 \
+                    macports-gcc-6 \
+                    macports-gcc-5 \
+                    macports-gcc-4.9 macports-gcc-4.8 macports-gcc-4.7
```

Comment:
Generally we disfavour using whitelists at all. Instead its better to blacklist all unwanted compilers, and then append those that are ok to the fallback list.

Reply:
Done.  Please check.